Resources/Prototypes/Entities/Clothing/Head/misc.yml

```diff
@@ -153,3 +153,15 @@
     sprite: Clothing/Head/Hats/catears.rsi
   - type: AddAccentClothing
     accent: OwOAccent
+
+- type: entity
+  parent: ClothingHeadBase
+  id: ClothingHeadHatCthulhu
+  name: Cthulhu Mask
```

Comment:
Trademark and IP for Cthulhu is... Difficult...
https://lovecraft.fandom.com/wiki/Copyright_status_of_works_by_H._P._Lovecraft

Maybe rename it something neutral, like "Elder God Mask" or "Squid Mask"?

Reply:
basically cthulhu himself is in the public domain as he was a creature created prior to the trademarking, the novel call of cthulhu however is not subject to public domain but the character is free domain. 

Cthulhu himself is very much in the public domain, as is the majority of Lovecraft's fiction. It's not copyrights that are the hazard here, it's trademarks. "Call of Cthulhu" is an established trademark in gaming, variously owned and licensed. Ditto for other game titles you might already know

Reply:
i changed the name of the item and description to a more generic Eldritch. I cannot however figure out how to change the .rsi file name from cthulhu to eldritch although it probably doesnt matter as much?

Reply:
> how to change the .rsi file name from cthulhu to eldritch

You just rename folder and path to it. Thats it.